types/debounce-promise/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for debounce-promise 3.1
+// Project: https://github.com/bjoerge/debounce-promise
+// Definitions by: Wu Haotian <https://github.com/whtsky>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+declare namespace debounce {
+    interface DebounceOptions {
+        leading?: boolean;
+        accumulate?: boolean;
+    }
+}
+
+type ArgumentsType<T> = T extends (...args: infer A) => any ? A : never;
```

Comment:
How is this different from `Parameters<T>`?

Reply:
Ah, I forgot about `Parameters<T>`. Thanks for pointing out.